### PR TITLE
module: MALT/BUILD

### LINKED ISSUE #636: new module: MALT
I think it would be good to have a module for [MALT(https://bioconda.github.io/recipes/malt/README.html)

- [x] This module does not exist yet with the [`nf-core modules list`](https://github.com/nf-core/tools#list-modules) command
- [x] There is no [open pull request](https://github.com/nf-core/modules/pulls) for this module
- [x] There is no [open issue](https://github.com/nf-core/modules/issues) for this module
- [x] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

### PR DESCRIPTION
**Pull request by jfy133**
Adds the MALT build module, required for MALT (run)

## PR checklist

Closes #636 

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `<SOFTWARE>.version.txt` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
    - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd`

### CODE REVIEW

File: modules/malt/build/main.nf [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+params.options = [:]
+options        = initOptions(params.options)
+
+process MALT_BUILD {
+
+    label 'process_high'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:[:], publish_by_meta:[]) }
+
```
Comment (grst) [inline modules/malt/build/main.nf:13]: Maybe worth adding a comment here for future reference, why you don't want the version to be bumped?

**Review by grst — APPROVED** [review]
Added one tiny comment, otherwise LGTM!

File: modules/malt/build/main.nf [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    conda (params.enable_conda ? "bioconda::malt=0.5.2" : null)
```
Comment (jfy133) [inline modules/malt/build/main.nf:14]:
```suggestion
    // Do not **auto-bump** due to problem with change of version numbering between 0.4.1 and 0.5.2 (originaly 0.4.1 was listed as 0.41, so is always selected as 'latest' even though it is
    conda (params.enable_conda ? "bioconda::malt=0.5.2" : null)
```

File: modules/malt/build/main.nf [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+options        = initOptions(params.options)
+
+process MALT_BUILD {
+
+    label 'process_high'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:[:], publish_by_meta:[]) }
+
+    // Do not **auto-bump** due to problem with change of version numbering between 0.4.1 and 0.5.2 (originaly 0.4.1 was listed as 0.41, so is always selected as 'latest' even though it is
```
Comment (jfy133) [inline modules/malt/build/main.nf:14]:
```suggestion
    // Do not **auto-bump** due to problem with change of version numbering between 0.4.1 and 0.5.2 
    // (originally 0.4.1 was listed as 0.41, so is always selected as 'latest' even though it is not!)
```

**Pull request merged**